<?xml version="1.0"?>
<launch>
  <arg name = "real_machine" default="True"/>
  <arg name = "switch_type" default="1"/>
  <arg name = "male_servo_id" default="5"/>
  <arg name = "female_servo_id" default="6" />
  <arg name = "unlock_servo_angle_male" default="7000"/>
  <arg name = "lock_servo_angle_male" default="8400"/>
  <arg name = "unlock_servo_angle_female" default="11000"/>
  <arg name = "lock_servo_angle_female" default="5800"/>
  <arg name = "left_edge_id" default="1"/>
  <arg name = "right_edge_id" default="3"/>

  <node pkg="beetle" type="switch_cmd_pub.py" name="" output="screen" >
    <param name = "real_machine" value="$(arg real_machine)"/>
    <param name = "switch_type" value="$(arg switch_type)"/>
    <param name = "male_servo_id" value="$(arg male_servo_id)"/>
    <param name = "female_servo_id" value="$(arg female_servo_id)"/>
    <param name = "unlock_servo_angle_male" value="$(arg unlock_servo_angle_male)"/>
    <param name = "lock_servo_angle_male" value="$(arg lock_servo_angle_male)"/>
    <param name = "unlock_servo_angle_female" value="$(arg unlock_servo_angle_female)"/>
    <param name = "lock_servo_angle_female" value="$(arg lock_servo_angle_female)"/>
    <param name = "left_edge_id" value="$(arg left_edge_id)"/>
    <param name = "right_edge_id" value="$(arg right_edge_id)"/>
  </node>
</launch>
